_episodes_rmd/02-r-basics.Rmd

```diff
@@ -726,7 +726,7 @@ c("ACTN3","APOA5") %in% snp_genes
 Lists are quite useful in R, but we won't be using them in the genomics lessons.
 That said, you may come across lists in the way that some bioinformatics
 programs may store and/or return data to you. One of the key attributes of a
-list is that unlike a vector, a list may contain data of more than one mode.
+list is that unlike a vector, a list may contain data of more than one class.
```

Comment:
I've always been confused about the distinction between data types, modes, and classes in R. However, the correct word here appears to be mode. (http://eriqande.github.io/rep-res-web/lectures/atomic-data-types-and-coercion.html#basic-modes)